diff --git a/BUILD b/BUILD
index 8ab4cb413..6746c9268 100644
--- a/BUILD
+++ b/BUILD
@@ -29,6 +29,10 @@ config_setting(
     },
 )
 
+ADDITIONAL_DEPS = [
+ #    "@org_llvm_libcxx//:libcxx",
+]
+
 GTEST = select({
     "//:use_com_google_googletest": [
         "@com_google_googletest//:gtest",
@@ -473,7 +477,7 @@ cc_binary(
     srcs = ["src/google/protobuf/compiler/main.cc"],
     linkopts = LINK_OPTS,
     visibility = ["//visibility:public"],
-    deps = [":protoc_lib"],
+    deps = [":protoc_lib"] + ADDITIONAL_DEPS,
 )
 
 ################################################################################
@@ -574,7 +578,7 @@ cc_binary(
     deps = [
         ":protobuf",
         ":protoc_lib",
-    ] + GTEST,
+    ] + GTEST + ADDITIONAL_DEPS,
 )
 
 cc_test(
@@ -765,7 +769,7 @@ cc_binary(
     deps = select({
         "//conditions:default": [],
         ":use_fast_cpp_protos": ["//external:python_headers"],
-    }),
+    }) + ADDITIONAL_DEPS,
 )
 
 cc_binary(
@@ -789,7 +793,7 @@ cc_binary(
     deps = [
         ":protobuf",
         ":proto_api",
-    ] + select({
+    ] + ADDITIONAL_DEPS + select({
         "//conditions:default": [],
         ":use_fast_cpp_protos": ["//external:python_headers"],
     }),
@@ -1176,7 +1180,7 @@ cc_binary(
         ":binary_json_conformance_suite",
         ":conformance_test",
         ":text_format_conformance_suite",
-    ],
+    ] + ADDITIONAL_DEPS,
 )
 
 sh_test(
